mobile/evidence/ios/Features/Sources/Login/LoginEmailViewModel.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  LoginEmailViewModel.swift
+//
+//
+//  Created by Cris Messias on 24/01/24.
+//
+
+import Foundation
+import Dependencies
+
+final public class LoginEmailViewModel: ObservableObject, Identifiable {
+    public var id = UUID()
+    @Published var emailInput: String
+    @Published var passwordInputMock: String
+    @Published var isValidEmail: Bool
+    @Published var isNextButtonPressed: Bool
+    @Published var isEmailInputFocused: Bool
+    @Published var loginCheckViewModel: LoginCheckEmailViewModel?
+    var delegateCloseButtonTapped: () -> Void = { fatalError() }
+    @Dependency(\.loginManager) private var loginManager
+
+    public init(
+        emailInput: String = "",
+        passwordInput: String = "123456",
+        isValidEmail: Bool = false,
+        isNextButtonPressed: Bool = false,
+        isInputEmailFocused: Bool = false,
+        loginCheckViewModel: LoginCheckEmailViewModel? = nil
+    ) {
+        self.emailInput = emailInput
+        self.passwordInputMock = passwordInput
+        self.isValidEmail = isValidEmail
+        self.isNextButtonPressed = isNextButtonPressed
+        self.isEmailInputFocused = isInputEmailFocused
+        self.loginCheckViewModel = loginCheckViewModel
+    }
+
+    func signIn() {
+        Task {
+            do {
+                let returnUserData = try await loginManager.creatUser(email: emailInput, password: passwordInputMock)
```

Comment:
Soooo. In this feature don't, but since will work on login with email & password I probably will do something with this data. 

What do you have in mind? hehehe